src/components/axes/timeAxis.ts

```diff
@@ -182,7 +182,16 @@ export module Axes {
       this.axisConfigurations(Time._DEFAULT_TIME_AXIS_CONFIGURATIONS);
     }
 
+    /**
+     * Gets the label positions for each tier.
+     */
     public tierLabelPositions(): string[];
+    /**
+     * Sets the label positions for each tier.
+     * 
+     * @param {string[]} newPositions The posititons for each tier. "bottom" and "center" are the only supported values.
```

Comment:
Not sure if "The posititons for each tier." is useful.

Also spelled wrong.

Consider the string array idea from above axis documentation